File: pueblo-golf-tournament-api/Dto/TournamentTeamDivisionDto.cs
using pueblo_golf_tournament_api.Base;

namespace pueblo_golf_tournament_api.Dto
{
    public class TournamentTeamDivisionDto : BaseDto
    {
        public long TeamId { get; set; }
        public long TournamentId { get; set; }
        public long DivisionId { get; set; }
    }
}

File: pueblo_golf_tournament_mobile/lib/dto/model/tournament-team-division-dto.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:pueblo_golf_tournament_mobile/base/base-dto.dart';

part 'tournament-team-division-dto.g.dart';

@JsonSerializable()
class TournamentTeamDivisionDto extends BaseDto {
  final int divisionId;
  final int teamId;
  final int tournamentId;
  TournamentTeamDivisionDto({
    required super.id,
    required this.divisionId,
    required this.teamId,
    required this.tournamentId,
  });

  factory TournamentTeamDivisionDto.fromJson(Map<String, dynamic> json) =>
      _$TournamentTeamDivisionDtoFromJson(json);

  Map<String, dynamic> toJson() => _$TournamentTeamDivisionDtoToJson(this);
}
